### PR TITLE
[IMP] account: get default payment ref format with country

### PR DESCRIPTION
**Pull request by william-andre**
Task [2297165](https://www.odoo.com/web#id=2297165&action=333&active_id=967&model=project.task&view_type=form&cids=1&menu_id=4720)

Set the payment reference standard to the one of the country of the
company by default.
The module l10n_be_invoice_bba has been merged into l10n_be as it was
auto_installed, it will not be problematic not being able to uninstall
it, and the order of installation of modules made it so that the
journals were created before l10n_be_invoice_bba was installed. This
caused the value to be set to 'odoo' on a fresh database instead of 'be'

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by william-andre** [comment]
@qdp-odoo Note that the `.po` files were completely outdated before saying anything about it :wink:

**Review by qdp-odoo — COMMENTED** [review]
doesn't fully work :thinking:

File: addons/account/models/account_journal.py [hunk 1]
```diff
@@ -39,6 +39,13 @@ def _get_bank_statements_available_sources(self):
     def _default_alias_domain(self):
         return self.env["ir.config_parameter"].sudo().get_param("mail.catchall.domain")
 
+    def _default_invoice_reference_model(self):
+        country_code = self.env.company.country_id.code
+        country_code = country_code and country_code.lower()
+        if country_code in self._fields['invoice_reference_model'].get_values(self.env):
```
[the remark this answers is not in the document]
Comment (william-andre) [reply addons/account/models/account_journal.py:45]: I wasn't even aware those existed :open_mouth:
Should I just pick the first one?

**Comment by william-andre** [comment]
@jco-odoo Why is `l10n_se_ocr` not in `l10n_se`?
Should I merge both modules?

**Comment by jco-odoo** [comment]
@william-andre There is a task for that already.

**Comment by william-andre** [comment]
@qdp-odoo Okay then, we can merge this, or I could take care of said task here?

File: addons/account/models/account_journal.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (qdp-odoo) [reply addons/account/models/account_journal.py]: like you did it's OK :+1:

**Comment by qdp-odoo** [comment]
@robodoo r+ rebase-ff

**Pull request closed without merging**